components/ParameterInfoCard.vue

```diff
@@ -0,0 +1,78 @@
+<template>
```

Comment:
This file doesn't contain new code (except a little tidying of the paramDisplayText function) - it's just been extracted wholesale from [runId].vue in order to make that component smaller.